recipe/meta.yaml

```diff
@@ -1,21 +1,21 @@
 {% set name = "pandas-profiling" %}
```

Comment:
The package name is now `ydata-profiling` https://github.com/ydataai/ydata-profiling/blob/305b251f8094528cb8a6fa6fb48378c024a7ff1a/setup.py#L23.
We need to fork and clone a new feedstock for it from conda-forge https://github.com/conda-forge/ydata-profiling-feedstock/blob/main/recipe/meta.yaml